tensorflow/core/platform/BUILD

```diff
@@ -1095,7 +1095,10 @@ cc_library(
     copts = tf_copts(),
     linkopts = select({
         "//tensorflow:windows": [],
-        "//conditions:default": ["-lm"],
+        "//conditions:default": [
+            "-lm",
+            "-ldl",
```

Comment:
The flag should probably be added to linkopts here instead: https://github.com/tensorflow/tensorflow/blob/71738a2bfb3824611debc6c54dc26c50072473a0/tensorflow/core/platform/default/BUILD#L376

Reply:
Makes sense to me too. Added a commit doing that. Quick test shows that this works.